parking_permits_app/models/address.py

```diff
@@ -27,6 +27,7 @@ class Address(TimestampedModelMixin):
     location = models.PointField(
         _("Location (2D)"), srid=settings.SRID, blank=True, null=True
     )
+    primary = models.BooleanField(_("Primary address"), default=False)
```

Comment:
It just came in my mind, but would it be better to use something like `is_primary`?